workflow/Snakefile

```diff
@@ -84,18 +84,19 @@ rule clean:
          print("Data downloaded to data/ has not been cleaned.")
 
 
-rule retrieve_ariadne_scenario_data:
+rule retrieve_ariadne_database:
     params:
         iiasa_usr=os.environ["IIASA_USERNAME"],
```

Comment:
I would add [iiasa_database][leitmodelle] and [iiasa_databse][scenarios] as parameters from the config.yaml